OpenRA.Mods.Common/Traits/Attack/AttackOmni.cs

```diff
@@ -29,6 +29,14 @@ public override Activity GetAttackActivity(Actor self, Target newTarget, bool al
 			return new SetTarget(this, newTarget);
 		}
 
+		public override void ResolveOrder(Actor self, Order order)
```

Comment:
What's the motivation for moving this from `AttackCharge` to `AttackOmni` (why not `AttackBase`)?